src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(" Figure: circle, area:" + getArea() + "sq. units, side: "
+                + radius + "units, color: " + color);
```

Comment:
The 'color' variable is not defined in the Circle class. It seems you intended to use the 'color' from the superclass 'Figure'. Ensure that 'color' is accessible here, possibly by using a getter method from the superclass if it's private.